### ISSUE TITLE
Implement PlayerCell controller and Player class

### ISSUE THREAD

**Issue by BryceDMonaco** [BryceDMonaco/DMTool#17]
Blocked by #16 
* Implement the Player class
* Implement the PlayerCell controller (setOnAction() for all parts which need it, set text to relevant variables from a given Player object.

Do not implement parsing from or saving to a save file, another issue exists for that.
